server/api/support.py

```diff
@@ -0,0 +1,171 @@
+import uuid
+import secrets
+from urllib.parse import urlparse
+from flask import jsonify, request, session, redirect
+from auth0.v3.authentication import GetToken
+from auth0.v3.management import Auth0
+from auth0.v3.exceptions import Auth0Error
+from werkzeug.exceptions import Conflict
+
+from . import api
+from ..models import *  # pylint: disable=wildcard-import
+from ..database import db_session
+from ..auth import (
+    restrict_access_superadmin,
+    set_loggedin_user,
+    UserType,
+)
+from ..config import (
+    AUDITADMIN_AUTH0_BASE_URL,
+    AUDITADMIN_AUTH0_CLIENT_ID,
+    AUDITADMIN_AUTH0_CLIENT_SECRET,
+)
+from ..util.jsonschema import validate
+
+AUTH0_DOMAIN = urlparse(AUDITADMIN_AUTH0_BASE_URL).hostname
+
+
+def auth0_get_token() -> str:
+    response = GetToken(AUTH0_DOMAIN).client_credentials(
+        AUDITADMIN_AUTH0_CLIENT_ID,
+        AUDITADMIN_AUTH0_CLIENT_SECRET,
+        f"https://{AUTH0_DOMAIN}/api/v2/",
+    )
+    return str(response["access_token"])
+
+
+def auth0_create_audit_admin(email: str):
+    token = auth0_get_token()
+    auth0 = Auth0(AUTH0_DOMAIN, token)
+    try:
+        auth0.users.create(
+            dict(
+                email=email,
+                password=secrets.token_urlsafe(),
+                connection="Username-Password-Authentication",
+            )
+        )
```

Comment:
should we store an `external_user_id`, or is the user keyed by `email` in the Auth0 DB?

Reply:
added here: 972b1d4